### DISCUSSION TITLE
hiding child  tables in separate schema

### DISCUSSION THREAD

**Discussion by scrappy** [pgpartman/pg_partman#423] [Q&A] [answered]
Subject pretty much says it all .. is there a method by which I can have the child tables stored in a separate schema from the parent?

for instance:

public.parent
  parent_schema.child

Its purely an aesthetics thing, but since dev folks are only going to need to reference parent, "hiding" the child tables would be nice ... if possible.

I tried creating schema, doing 'set search_path' into said schema before running create_parent(), but that didn't work, and am not seeing any option to create_parent to suggest it is possible, but figured I'd ask before I write that off ...

thx

**Comment by keithf4** [comment, marked as answer]
I've tried handling child tables in different schemas in pg_partman before and it made things really overly complicated. And most of the time it really was just for aesthetic reasons, so I couldn't really justify the complication for that.

Perhaps if trigger-based partitioning is dropped next year, I can revisit this since that will remove a tremendous amount of complexity in this extension. Sorry I can't offer help on it at this time though.

**Comment by scrappy** [comment]
Not a problem, thx for the update .. mostly wanted to make sure there wasn't something I was missing that would allow me to do it.
